openapi.yaml

```diff
@@ -431,7 +431,8 @@ components:
     order_statuses:
       type: object
       required:
-        - conformsTo
+        - current
+        - history

```

Comment:
Update to conform to: https://github.com/stapi-spec/stapi-spec/pull/239/files#diff-d910ba2ef878f7db0223a966b81c8b3f3b65027bb39e4431bb05140171eece39